hack/verify-govet.sh

```diff
@@ -29,8 +29,10 @@ function govet_blacklist_contains() {
 	return 1
 }
 
+govet_args="-shadow=false -printfuncs=Info,Infof,Warning,Warningf"
+
 for test_dir in $(os::util::list_go_src_dirs); do
-	if ! result="$(go tool vet -shadow=false "${test_dir}" 2>&1)"; then
+	if ! result="$(go tool vet ${govet_args} "${test_dir}" 2>&1)"; then
```

Comment:
If you're using them once, just inline them here

Reply:
personally i like the way it's called out as a var, makes it easier to see the args and update them rather than embedding them in a messy line of bash.

Reply:
oops, just pushed a change. i'm fine either way

Reply:
no need to churn on it so i guess we'll leave it.